src/main/java/core/basesyntax/service/FileReader.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service;
+
+import java.util.List;
+
+public interface FileReader {
+    List<String> read();
```

Comment:
Consider adding a parameter to the `read` method for specifying the file path. This will make the interface more flexible and reusable for different files.